woocommerce/woocommerce-ios
# Polish footer spinner view for `plain` table view style

Fixes #1262 

Update release notes:


## Changes
- Added `tableViewStyle: UITableView.Style` to `FooterSpinnerView` initializer
- Updated `FooterSpinnerView` usage in Orders, Oders Search, and Shipment Providers (all of these view controllers have grouped table view style)
- Updated `FooterSpinnerView` usage in `ProductsViewController`, and used `FooterSpinnerView`/empty footer view based on spinner state

## Testing
- Log out of the app and then log in to a store with products
- Go to Products tab
- Scroll down the product list and observe the spinner at the bottom of the table view --> the spinner should be horizontally centered with space at the top and bottom
- After the products are all loaded, scroll to the bottom --> there should be no spacing at the footer

## Example screenshots

- Products tab

\ | before | after
-- | -- | --
with spinner | <img width="549" alt="Screen Shot 2019-09-17 at 10 46 49 AM" src="https://user-images.githubusercontent.com/1945542/65066622-8c329800-d939-11e9-8157-abb6f72e16dc.png"> | <img width="549" alt="Screen Shot 2019-09-30 at 11 09 40 AM" src="https://user-images.githubusercontent.com/1945542/65847118-d45aad00-e372-11e9-9b23-fbe3ed5cdb27.png">
without spinner | <img width="549" alt="Screen Shot 2019-09-17 at 10 46 12 AM" src="https://user-images.githubusercontent.com/1945542/65066619-8c329800-d939-11e9-84ee-a701c324bf05.png"> | <img width="549" alt="Screen Shot 2019-09-17 at 10 56 39 AM" src="https://user-images.githubusercontent.com/1945542/65066791-f1868900-d939-11e9-8c29-a43da62cf4f2.png">

- Order Search

before | after
-- | --
![Simulator Screen Shot - iPhone 5s - 2019-09-30 at 11 12 16](https://user-images.githubusercontent.com/1945542/65847218-34e9ea00-e373-11e9-8ac0-b84c261b16bd.png) | ![Simulator Screen Shot - iPhone 5s - 2019-09-30 at 10 46 23](https://user-images.githubusercontent.com/1945542/65847180-09ff9600-e373-11e9-9f27-f3681040b6c3.png)
